### PR TITLE
PHP_Task

### PR DESCRIPTION
**Pull request by hummingbed**
### Task: <task name here>

I <clear and concise explanation>
I <clear and concise explanation>

### CODE REVIEW

**Comment by hummingbed** [comment]
PHP_TASK

File: submissions/testfile.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this is a test file
```
Comment (adaugochi) [inline submissions/testfile.txt]: What's this file for? Please remove it when not in use

**Comment by ndanni** [comment]
Kindly rename this Pull request to PHP task please.

**Comment by hummingbed** [comment]
I just renamed to PHP Task as requested

File: submissions/.gitignore [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Ignore everything in this directory
-*
+
```
Comment (adaugochi) [inline submissions/.gitignore]: You aren't supposed to push this file. Please revert the changes made here

File: submissions/MichaelMenebraye/PHP_Task/Input_Dashboard.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+}
+while($row = $stmt->fetch(PDO::FETCH_OBJ)){
+    echo '  <tr>
+                <th> '.$row->id.'</th>
+                <th> '.$row->brands.'</th>
+                <th> '.$row->brand_name.'</th>
+                <th> '.$row->quantity.'</th>
+                <th> '.$row->price.'</th>
+                <th> '.$row->date.'</th>
+    ';
```
Comment (adaugochi) [inline submissions/MichaelMenebraye/PHP_Task/Input_Dashboard.php]: No closing `</tr>` tag

**Review by beccadaniel — CHANGES_REQUESTED** [review]
Add the README.md file back. You don't have to make any changes to this particular file.

**Comment by hummingbed** [comment]
Noted, comments effected

**Review by adaugochi — APPROVED** [review]

**Review by beccadaniel — APPROVED** [review]

**Pull request merged**